template/web/src/components/SmartWallets/PaymasterClient.ts

```diff
@@ -1,7 +1,16 @@
-import { createPublicClient, http } from 'viem';
+import { ENTRYPOINT_ADDRESS_V06 } from 'permissionless';
+import { paymasterActionsEip7677 } from 'permissionless/experimental';
+import { createClient, createPublicClient, http } from 'viem';
 import { baseSepolia } from 'viem/chains';
 
+const paymasterService = process.env.NEXT_PUBLIC_PAYMASTER_URL ?? '';
```

Comment:
Was this in the config?

Reply:
This was originally in the `constants.ts` file. I figured that it wasn't necessary to assign a const to the env variable when I can just access the env whenever I needed it. 

It looked like:
`const PAYMASTER_SERVICE_URL = process.env.NEXT_PUBLIC_PAYMASTER_URL`